versioned_docs/version-8.0/apis-tools/operate-api/index.md

```diff
@@ -3,7 +3,6 @@ id: index
 title: Operate API (REST)
 description: "Operate API is a REST API and provides searching, getting, and changing Operate data.
 Requests and responses are in JSON notation."
-canonicalUrl: "/docs/apis-tools/operate-api/overview"
```

Comment:
I _think_ this is okay to do. The canonical logic I introduced doesn't allow for fully-qualified URLs, so I can't point this at the supported version of this page. But this unsupported site is supposed to be un-indexed, so I think removing the canonical should be fine.

Let me know what you think! I can also add in logic to the canonical stuff to support pointing at the supported site.

Reply:
I'm following your logic here with similar thinking. I don't have strong opinions about making a change to point at the supported site.